### PR TITLE
Added APIM_WORKFLOW.siddhi

### PR DESCRIPTION
**Pull request by shilmyhasan**
Added APIM_WORKFLOW.siddhi to apim-analytics-feature

### CODE REVIEW

**Comment by this** [comment]
@shilmyhasan In future commits, please limit the commit message to 50 character. We follow this guideline https://chris.beams.io/posts/git-commit/

File: features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_WORKFLOW.siddhi [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+@App:name("APIM_WORKFLOW")
+@App:description("Workflow Stream Definition")
+
+@source(type = 'wso2event', wso2.stream.id = 'org.wso2.apimgt.statistics.workflow:1.0.0', 
+	@map(type = 'wso2event'))
+define stream workflowStream(
```
Comment (grainier) [inline features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_WORKFLOW.siddhi]: Is this contain only the stream definition?

Comment (shilmyhasan) [reply features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_WORKFLOW.siddhi]: yes

Comment (grainier) [reply features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_WORKFLOW.siddhi]: what's the purpose?

Comment (shilmyhasan) [reply features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_WORKFLOW.siddhi]: it was kind of added if someone wants to create custom implementations related to workflows. APIM sends events for this stream (in previous versions also it was done).

**Pull request merged**